package.json

```diff
@@ -33,6 +35,9 @@
         "prettier": "2.3.2",
         "rimraf": "^3.0.2",
         "salesforce-pageobjects": "^1.1.0",
+        "salesforceapp-pageobjects": "^236.1.1",
```

Comment:
once the salesforce-pageobjects includes the mobile pos, will remove this one

Reply:
Is `salesforceapp-pageobjects` published externally though? Is it on npm or only internally in Nexus?

Reply:
yes, this is internal only. Will not merge this until salesforce-pageobjects includes the mobile pos. Just use this one here to show how to demonstrate how to use it in test.

Reply:
Sounds good! thank you for the explanations! I will resolve this conversation